.github/sync.yml

```diff
@@ -28,8 +28,8 @@ group:
         dest: .github/actionlint-matcher.json
       - source: .github/workflows/subscribe_to_label.yml
         dest: .github/workflows/subscribe_to_label.yml
```

Comment:
Shouldn't these be updated as well?

Reply:
Good eye! Those are the workflow names (which differ from the data file at `.github/subscribe-to-label.json`), which have not been changed: https://github.com/WordPress/openverse/blob/main/.github/workflows/subscribe_to_label.yml

So we're good there 🙂